docs/migration-guide.md

```diff
@@ -58,171 +82,63 @@ To add Storybook to a project that isn’t currently using Storybook:
-};
-```
-
-```js
-// .storybook/main.js
-
-module.exports = {
-  stories: ['../src/**.stories.@(js|jsx|mjs|ts|tsx)'],
-};
-```
-
-#### After
+You’ll also need to update your stories glob in .storybook/main.js to include the newly created .mdx and .stories.js files if it doesn’t already.
 
-Your MDX files should be updated to reference the CSF story export, which is safer and cleaner than referencing the string ID:
+**Note:** this migration supports the Storybook 6 [“CSF stories with MDX docs”](https://github.com/storybookjs/storybook/blob/6e19f0fe426d58f0f7981a42c3d0b0384fab49b1/code/addons/docs/docs/recipes.md#csf-stories-with-mdx-docs) recipe.
 
-<!-- prettier-ignore-start -->
-```mdx
-{/* Button.mdx */}
-
-{/* Note the new package we import from */}
-import { Story } from '@storybook/blocks';
-import * as ButtonStories from './Button.stories';
-
-# Story embed
+## Troubleshooting
 
-<Story of={ButtonStories.Basic} />
-```
-<!-- prettier-ignore-end -->
+The automatic upgrade should get your Storybook into a working state. If you encounter an error running Storybook after upgrading, here’s what to do:
 
-You should also remove the MDX imports from your story files:
+1. Try running the [`doctor` command](./api/cli-options.md#doctor) to check for common issues (such as duplicate dependencies, incompatible addons, or mismatched versions) and see suggestions for fixing them.
+2. If you’re running `storybook` with the `dev` command, try using the `build` command instead. Sometimes `build` errors are more legible than `dev` errors!
+3. Check [the full migration notes](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#from-version-7x-to-800), which contains an exhaustive list of noteworthy changes in Storybook 8. Many of these are already handled by automigrations when you upgrade, but not all are. It’s also possible that you’re experiencing a corner case that we’re not aware of.
+4. Search [Storybook issues on GitHub](https://github.com/storybookjs/storybook/issues). If you’re seeing a problem, there’s a good chance other people are too. If so, upvote the issue, try out any workarounds described in the comments, and comment back if you have useful info to contribute.
+5. If there’s no existing issue, you can [file one](https://github.com/storybookjs/storybook/issues/new/choose), ideally with a reproduction attached. We’ll be on top of Storybook 8 issues as we’re stabilizing the release.
 
-```js
-// Button.stories.js
+If you prefer to debug yourself, here are a few useful things you can do to help narrow down the problem:
 
-import { Button } from './Button';
+1. Try removing all addons that are not in the `@storybook` npm namespace. Community addons that work well with 7.x might not yet be compatible with 8.0, and this is the fastest way to isolate that possibility. If you find an addon that needs to be upgraded to work with Storybook 8, please post an issue on the addon’s repository, or better yet, a PR to upgrade it!
```

Comment:
the only package which doesn't fall into the `@storybook` namespace which people should never remove is `storybook`, which powers the CLI!